src/irr/asset/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -90,6 +90,18 @@ SAssetBundle CGraphicsPipelineLoaderMTL::loadAsset(io::IReadFile* _file, const I
     {
         auto layout = makePipelineLayoutFromMtl(materials[i]);
         pipelines[i] = core::make_smart_refctd_ptr<ICPURenderpassIndependentPipeline>(nullptr, std::move(layout), nullptr, nullptr, vtxParams, blendParams, primParams, rasterParams);
+        if (materials[i].maps[CMTLPipelineMetadata::SMtl::EMP_OPACITY].size())
```

Comment:
doesn't MTL support like way more ways to make stuff blendable? or is it only the opacity map?

Reply:
theres also constant opacity parameter, actually i could also taken into account if constant opacity `!=1.0`

Reply:
done

Reply:
what about a thing called "transmission" or something like that which I saw in the structs/image map definitions?

Reply:
yea, i wrongly understood this "transmission filter" before (now it's not used anyway though). Look at `8  Real windshield` at http://paulbourke.net/dataformats/mtl/ . I don't really understand what does it mean that "the material is not dissolved" but "filters of 
colorizes objects that are seen through it" at the same time. Should assume that the material is fully transparent as soon as `Tf` is defined in MTL, but use some constant non-zero alpha for blending framebuffer color with `Tf` (transmission filter) color?

Reply:
well that spec is so badly written I have no idea what it was supposed to stand for.

My guess would be some sort of attenuation value per-unit-thickness-of-object.

So you're bascailly looking at
```glsl
outColor.rgb = mix(src.rgb,dst.rgb,exp2(unitConversionConstant*Tf*thickness));
```
It would be possible to implement with "dual source blending", but obviously no idea what the resultant alpha should be.

Since its impossible to render in a raster pipeline and only using a "sane" 3 or 4 channel single attachment framebuffer, we shoudl issue a warning that this parameter and light model has been detected, fall-back to something useful, but still populate the descriptor set with the correct image/texture and provide as much data in the metadata as possible.

Reply:
did you implement the warning and fallback?

Reply:
yea, in upcoming commit